### PR TITLE
CSS files: fix coding standard violations #7931

### LINKED ISSUE #7931: CSS files: fix coding standard violations
Some of the CSS files are not following the CSS Coding Standard as defined by TEAMMATES [here](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub).
***
| | File Name  | Following Coding Standard not Followed  |
| ------------ | ------------ | ------------ |
| :white_large_square: | [adminAccountDetails.css](https://github.com/TEAMMATES/teammates/blob/master/src/main/webapp/stylesheets/adminAccountDetails.css)  | [Naming Standards](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub#h.8aauk3v0ixn6), [Order of attributes](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub#h.cdhasngawcmb)  |
| :white_large_square: | [datepicker.css](https://github.com/TEAMMATES/teammates/blob/master/src/main/webapp/stylesheets/datepicker.css)  |    [Order of attributes](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub#h.cdhasngawcmb) |
| :white_large_square: | [instructorCourseEnroll.css](https://github.com/TEAMMATES/teammates/blob/master/src/main/webapp/stylesheets/instructorCourseEnroll.css)  | [Naming Standards](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub#h.8aauk3v0ixn6), [Order of attributes](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub#h.cdhasngawcmb)  |
| :white_large_square: | [omniComment.css](https://github.com/TEAMMATES/teammates/blob/master/src/main/webapp/stylesheets/omniComment.css)  | [Order of attributes](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub#h.cdhasngawcmb), [Hexadecimal Notation](https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation)  |
| :white_check_mark: | [printview.css](https://github.com/TEAMMATES/teammates/blob/master/src/main/webapp/stylesheets/printview.css "printview.css")  | **Looks good** |
| :white_large_square: | [teammatesCommon.css](https://github.com/TEAMMATES/teammates/blob/master/src/main/webapp/stylesheets/teammatesCommon.css "teammatesCommon.css")  | [Naming Standards](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub#h.8aauk3v0ixn6), [Order of attributes](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub#h.cdhasngawcmb), [Hexadecimal Notation](https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation)  |
***

### PR DESCRIPTION
**Pull request by tshradheya**
Fixes #7931 

Fixed CSS coding style by following [TEAMMATES CSS Coding Standard](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub)

Work ongoing

**Outline of Solution**

1. Followed naming conventions for e.g. changed `color_orange` to `color-orange`
2. ~Fixed Selector formatting by specifying them in separate lines~

### CODE REVIEW

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 1]
```diff
@@ -780,7 +780,8 @@ Prevents content overflow in navbar. */
     width: 30px;
 }
 
-.ui-datepicker-calendar .ui-state-default {
+.ui-datepicker-calendar,
+.ui-state-default {
```
Comment (wkurniawan07) [inline src/main/webapp/stylesheets/teammatesCommon.css]: I'll just leave a warning here that the comma makes a semantic difference and not just syntactic.

**Comment by tshradheya** [comment]
@wkurniawan07 There is `.questionType` in line [1156](https://github.com/TEAMMATES/teammates/pull/8352/files#diff-640ee63152be08488b22c4f8192c8cf2L1156) and on doing a search I could not find any `class="questionType" . So will it be safe to remove that css attribute?

**Comment by wkurniawan07** [comment]
@tshradheya I'm going to break the bad news: it may well be used in some JS files (some of the jQuery scripts target CSS selectors) or some Java test files (some of the Selenium test scripts target CSS selectors).
Also, if `class="questionType"` is the way you look for instances, you're going to miss all `class="other-class questionType"`.

**Comment by tshradheya** [comment]
By searching for just `questionType` in the entire project, I could find just the variable `questionType` of `FeedbackQuestion` which is mainly in the .java files and .json ( testing files)

**Comment by wkurniawan07** [comment]
@tshradheya if that's the case then it should be safe to remove.

**Comment by tshradheya** [comment]
Ready for Review

**Comment by wkurniawan07** [comment]
@tshradheya I'm ok to accept this patch as is, but is this all you want to do for now? `teammatesCommon.css` still contains a lot of items styled `likeThis`.

Also, your PR title should be an exact match of the linked issue, not just some random message (presumably from your first commit message) appended with the issue number. This applies to your other PRs as well.

**Comment by tshradheya** [comment]
Ready for Review

**Comment by tshradheya** [comment]
Reminder: This is Ready for Reviewing

**Pull request merged**